homeassistant/components/hunterdouglas_powerview/button.py

```diff
@@ -48,6 +48,13 @@ class PowerviewButtonDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
         press_action=lambda shade: shade.jog(),
     ),
+    PowerviewButtonDescription(
+        key="favorite",
+        name="Favorite",
+        icon="mdi:heart",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        press_action=lambda shade: shade.favorite(),
+    ),
```

Comment:
This doesn't look like a related change?

Reply:
I see you added it to the docs so it looks like its to support the new shade type